blog/2022-07-15-typescript-1.0.0.md

```diff
@@ -0,0 +1,157 @@
+JavaScript APIs, such as getting the current time or a random number, have been replaced with [deterministic
+versions](https://docs.temporal.io/typescript/determinism#sources-of-non-determinism).
+
+With all of this put together, we eliminated an entire class of footguns and made it easier to get started with
+Temporal.
+
+## Built with the community
+
+When we released the first alpha version of the SDK back in March of 2021, we set out to iterate on the public API and
+have asked our community of users to help us shape it.
+
+We've been very fortunate to have gotten such wide adoption for an SDK at such an early stage. The SDK's Slack channel
+([#typescript-sdk](https://temporal.io/slack)) has grown to more than 1000 members, and we've been actively responding
+to and supporting users on a daily basis.
+
+The feedback and trust we've gotten from our early adopters who put the SDK in production has been invaluable. We
+wouldn't have been able to reach API and functional stability without them. Thank you for putting up with our [many
+breaking changes](https://github.com/temporalio/sdk-typescript/blob/main/CHANGELOG.md) during the alpha and beta. We are
+committed to avoiding backward-incompatible changes from now on.
+
+A special thank you to everyone [who contributed](https://github.com/temporalio/sdk-typescript/graphs/contributors) to
+the SDK's development—[Sushisource](https://github.com/Sushisource), [vkarpov15](https://github.com/vkarpov15),
+[mjameswh](https://github.com/mjameswh), [JoshuaKGoldberg](https://github.com/JoshuaKGoldberg),
+[yoDon](https://github.com/yoDon), [SamSokolin](https://github.com/SamSokolin),
+[julianocomg](https://github.com/julianocomg), [vitarb](https://github.com/vitarb),
+[joebowbeer](https://github.com/joebowbeer), [andreasasprou](https://github.com/andreasasprou), and
+[jameslnewell](https://github.com/jameslnewell)—and to [swyx](https://twitter.com/swyx) for writing and recording most
+of the [docs and tutorials](https://docs.temporal.io/typescript/introduction/) and tirelessly advocating for API
+simplication. We welcome more involvement from anyone, from helping answer community questions to submitting issues or
+PRs—check out [CONTRIBUTING.md](https://github.com/temporalio/sdk-typescript/blob/main/CONTRIBUTING.md) for more
+information.
+
+## The journey
+
+Since the first release of the SDK, we've made significant changes to the public API.
+
+We went from Workflows that look like this, where a Workflow interface was required, we relied on path aliases, and the
+Workflow name was based on the containing file—
+
+```ts
+import { Example } from '@interfaces/workflows';
+import { greet } from '@activities/greeter';
+
+async function main(name: string): Promise<string> {
+  const greeting = await greet(name);
+  console.log(greeting);
+}
+
+export const workflow: Example = { main };
+```
+
+—to this, where Activities are proxied using their types and Workflows names are function names—
```

Comment:
I would more clearly label these example snippets, maybe with timestamps to show they are a timeline.

Reply:
Yeah, I was thinking about this too